.internal-ci/helm/fog-ingest/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Can we add a shared `.helmignore` in the parent dir `helm/` ?

Reply:
We run the helm builds with the target of the package directory.  I'd like to keep the helmignore file in its standard place.